WordPress/WordPressUITests/WPUITestCredentials.swift

```diff
@@ -3,7 +3,8 @@ struct WPUITestCredentials {
     static let testWPcomUserEmail: String = "e2eflowtestingmobile@example.com"
     static let testWPcomUsername: String = "e2eflowtestingmobile"
     static let testWPcomPassword: String = "mocked_password"
-    static let testWPcomSiteAddress: String =  "e2eflowtestingmobile.wordpress.com"
+    static let testWPcomSiteAddress: String =  "infocusphotographers.wordpress.com"
```

Comment:
@rachelmcr – I had to adjust this in order to make it work – trying to log in without a `.wordpress.com` address uses the self-hosted code which, which isn't set up for mocking. It doesn't _really_ matter what this value is, as long as it's `something`.wordpress.com, but I figured keeping it in sync would be good?

The `testWPcomSiteDisplayName` prop is then used for UI tests to match what comes back from the updated mocks (which needs to look pretty).

Reply:
Makes sense to me. Ideally the self-hosted flow would also be fully mocked, but we can revisit that separately when we have more bandwidth for it. I opened https://github.com/wordpress-mobile/WordPress-iOS/issues/13234 to track that enhancement.